src/tools/clippy/clippy_lints/src/manual_float_methods.rs

```diff
@@ -83,7 +83,7 @@ impl Variant {
 impl<'tcx> LateLintPass<'tcx> for ManualFloatMethods {
     fn check_expr(&mut self, cx: &LateContext<'tcx>, expr: &'tcx Expr<'tcx>) {
         if !in_external_macro(cx.sess(), expr.span)
-            && (!cx.param_env.is_const() || cx.tcx.features().active(sym!(const_float_classify)))
```

Comment:
This `cx.param_env.is_const()` needs an alternative.

Reply:
Oh yeah I should check if this is a const context